crt_portal/cts_forms/migrations/0061_load_zips.py

```diff
@@ -1,13 +1,24 @@
+import boto3
+from botocore import UNSIGNED
+from botocore.client import Config
+import csv
 import logging
 import os
-from csv import DictReader
+
 
 from django.conf import settings
 from django.db import migrations, models
 
 logger = logging.getLogger(__name__)
 
-circle = os.environ.get('CIRCLE', False)
+
+def fetch_file():
+    # no credentials needed to pull from the public bucket
+    s3 = boto3.resource('s3', config=Config(signature_version=UNSIGNED), region_name='us-gov-west-1')
+    # this is a public link in the dev s3 bucket
+    bucket = s3.Bucket(u'cg-b2626a97-5c90-4d35-a39b-24f3b035d5c9')
```

Comment:
Not a show stopper but there's an opportunity to define this in settings from an environment variable

Reply:
We would want the same variable in Prod, Staging, Dev and Local. There is one document that is the source of truth so we want everyone pointing there.  Prod, Staging and local don't have this variable defined and this is the only place that we will be using it. 

In another PR I remove the zip loading from tests and that speeds each run up by a minute. This will only be run for a brand new database, say developer on-boarding or if they want to make a new enviornment in a sandbox or in feature etc.